rds-deployment-HA/azuredeploy.json

```diff
@@ -0,0 +1,397 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "dnsLabelPrefix": {
+      "type": "string",
+      "metadata": {
+        "description": "Unique public DNS prefix for the deployment. The fqdn will look something like '<dnsname>.westus.cloudapp.azure.com'. Up to 62 chars, digits or dashes, lowercase, should start with a letter: must conform to '^[a-z][a-z0-9-]{1,61}[a-z0-9]$'."
+      }
+    },
+    "loadBalancer": {
+      "type": "string",
+      "metadata": {
+        "description": "The Loadbalancer name must match from the RDS deployment. And default value taken by template is loadbalancer."
+      },
+      "defaultValue": "loadBalancer"
+    },
+    "backendAddressPools": {
```

Comment:
We have added the README file accordingly. Please let us know if you want us to make any changes to README file.